nav-app/src/app/matrix/technique-cell/contextmenu/contextmenu.component.html

```diff
@@ -59,5 +59,13 @@
                 {{contextMenuItem.label}}
             </div>
         </div>
+        <div *ngIf="links.length" class="contextMenu-section">
+            <div class="link-container" *ngFor="let link of links">
+                <div *ngIf="!link.divider" class="contextMenu-button" (click)="openLink(link)">
+                    {{link.label}}
+                </div>
+                <div class="divider" *ngIf="link.divider"></div>
```

Comment:
This works, but *ngIf also supports else blocks which can help simplify code in some cases. Probably not worth changing here though.